.github/workflows/tests.yml

```
@@ -34,16 +37,13 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install -U pip
-          python -m pip install -U wheel
-          python -m pip install -U tox
+          pip install -U wheel tox
```

I think we can skip wheel, but let's make sure pip is latest.
```suggestion
          pip install -U pip
          pip install -U tox
```

---

Why reinstall pip?